caffeine/src/main/java/com/percolate/caffeine/ViewUtils.java

```diff
@@ -238,19 +241,49 @@ public static void hideView(Activity context, int id) {
     }
 
     /**
+     * Sets visibility of the given view to <code>View.GONE</code>.
+     *
+     * @param parentView The View used to call findViewId() on.
+     * @param id         R.id.xxxx value for the view to hide "expected textView to throw a ClassCastException" + textView.
+     */
+    public static void hideView(View parentView, int id) {
+          View view = parentView.findViewById(id);
```

Comment:
`context` versions check if `context != null`,  Let's do the same in these 2 methods with `parentView`.